app/controllers/owners_controller.rb

```diff
@@ -10,19 +10,8 @@ def show
     # Only do this once
     session[:new_supporter] = false if @new_supporter
 
-    @scrapers = @owner.scrapers.includes(:last_run)
-
-    # Split out scrapers into different groups
-    @running_scrapers, @erroring_scrapers, @other_scrapers = [], [], []
-    @scrapers.each do |scraper|
-      if scraper.running?
-        @running_scrapers << scraper
-      elsif scraper.requires_attention?
-        @erroring_scrapers << scraper
-      else
-        @other_scrapers << scraper
-      end
-    end
+    @scrapers = @owner.scrapers.order(:updated_at).includes(:last_run)
```

Comment:
These lists were divided in https://github.com/openaustralia/morph/commit/4afea5d3fe683f03ceb6969d14b164849154465a and https://github.com/openaustralia/morph/commit/4bd3911351144b65cbc8b0a7ce9cabfacae06ef0 to make the scrapers people are most likely interested in more prominent. @snoblenet could you explain how merging the lists relates to this issue and helps users?